ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactStylesDiffMap.java

```diff
@@ -22,11 +22,12 @@
  * some of the accessor methods of {@link ReadableMap} by adding a default value property
  * such that caller is enforced to provide a default value for a style property.
  *
- * Instances of this class are used to update {@link View} or {@link CSSNode} style properties.
- * Since properties are generated by React framework based on what has been updated each value
- * in this map should either be interpreted as a new value set for a style property or as a "reset
- * this property to default" command in case when value is null (this is a way React communicates
- * change in which the style key that was previously present in a map has been removed).
+ * Instances of this class are used to update {@link View} or {@link ReactShadowNode} style
```

Comment:
Reflow to make this <=100 chars per line. The change is `s/CSSNode/ReactShadowNode`.